<launch>
  <param name="hough_thr" value="30  "/>
  <param name="min_lin_len" value="20" />
  <param name="max_gap_len" value="20" />
  <param name="value_thr_low" value="100"/>
  <param name="value_thr_high" value="255" />
  <param name="canny_thr_low" value="40" />
  <param name="canny_thr_high" value="75" />
 <node pkg="cross" type="cross_node" name="cross_node" output="screen">
   </node>
</launch>
